src/main/kotlin/no/nav/tiltakspenger/saksbehandling/behandling/infra/route/dto/BehandlingDTO.kt

```diff
@@ -115,6 +117,8 @@ fun Søknadsbehandling.tilSøknadsbehandlingDTO(): SøknadsbehandlingDTO {
         barnetillegg = null,
         valgteTiltaksdeltakelser = null,
         avslagsgrunner = null,
+        automatiskSaksbehandlet = this.automatiskSaksbehandlet,
+        manueltBehandlesGrunner = this.manueltBehandlesGrunner.map { it.visningsnavn },
```

Comment:
Frontenden vil nok ha kodene også, det er ikke så fleksibelt og måtte endre i backenden bare for å endre visning i frontend. For feilkoder har vi vel en hybrid der vi sender med begge deler også vi meldingen, hvis frontenden ikke har lagt inn en oversettelse selv.

Reply:
Da synes jeg vi kun skal ta vare på kodene og så la tekstene ligge i frontend, så vi slipper å vedlikeholde dem to steder :)